### ISSUE TITLE
@twurple/pubsub cannot find module @d-fischer/isomorphic-ws or its corresponding type declarations

### ISSUE THREAD

**Issue by TheAppleFreak** [twurple/twurple#334]
### Code

```ts
import { PubSubClient } from "@twurple/pubsub";
```

### Expected behavior

The package imports correctly and works properly in TypeScript.

### Actual Behavior

The TypeScript compiler outputs the following:

`node_modules/@twurple/pubsub/lib/BasicPubSubClient.d.ts(2,36): error TS2307: Cannot find module '@d-fischer/isomorphic-ws' or its corresponding type declarations.`

Adding `@d-fischer/isomorphic-ws` to `package.json` fixes the issue.

### Environment

- Version: 5.0.17
- Node version: 16.14.0
- Operating system: Alpine Linux (Docker)

**Comment by rodrigograca31** [comment]
I can confirm this is a current bug.

@d-fischer make sure you dont have `@d-fischer/isomorphic-ws` globally installed or something.

Im doing an upgrade from 4 to 5, just installed 
```json
"@twurple/api": "^5.0.17",
"@twurple/auth": "^5.0.17",
"@twurple/pubsub": "^5.0.17",
```

and get that error.

looking into the code tells me that `node_modules/@twurple/pubsub/lib/BasicPubSubClient.d.ts:2:36` imports `import type { ClientOptions } from '@d-fischer/isomorphic-ws';`but that package `package.json` doesnt include it as seen here:

https://github.com/twurple/twurple/blob/a1dc8535c5c023e240b9393fa088a7e27ef9d72f/packages/pubsub/package.json#L32-L39

**Comment by rodrigograca31** [comment]
Temporary fix:
Install the given package as a direct dependacy until this gets gixed:
`npm i @d-fischer/isomorphic-ws`

**Comment by rodrigograca31** [comment]
and thats one more reason why yarn sucks 😂 :hand_over_mouth:

**Comment by d-fischer** [comment]
I couldn't reproduce this with npm either, but I have still fixed up the dependency links and released the new version 5.0.18 that includes this fix.

**Comment by rodrigograca31** [comment]
I was just trying to make a reproducable example but cant....

I could explain why but its fixed. doesnt matter anymore...
